layers/+lang/c-c++/packages.el

```diff
@@ -315,3 +314,8 @@
             projectile-project-root-files-top-down-recurring))))))
 
 ;; END LSP BACKEND PACKAGES
+
+(defun c-c++/pre-init-dap-mode ()
+  (dolist (mode c-c++-modes)
+    (add-to-list 'spacemacs--dap-supported-modes mode)
+    (when (eq c-c++-backend 'lsp) (spacemacs/add-to-hooks 'spacemacs/c-c++-setup-lsp-dap c-c++-mode-hooks))))
```

Comment:
This function is written with double slash: spacemacs//c-c++-setup-lsp-dap

Reply:
...ahem... :)
Thanks @gessen -- fixed/pushed.